src/file.cpp

```diff
@@ -80,18 +80,18 @@ namespace Sass {
     bool file_exists(const std::string& path)
     {
       #ifdef _WIN32
-        wchar_t resolved[32768];
+        std::vector<wchar_t> resolved(32768);
```

Comment:
Perhaps we want to initialize with zeroes explicitly instead of expecting the allocator to do so?

```suggestion
        const std::size_t max_chars = 32767;
        std::vector<wchar_t> resolved(max_chars + 1, 0);
```